src/ring-call-channel.c

```diff
@@ -544,6 +553,15 @@ ring_call_channel_class_init(RingCallChannelClass *klass)
       G_PARAM_READABLE | G_PARAM_STATIC_STRINGS));
 
   g_object_class_install_property(
+    object_class, PROP_IMSI,
+    g_param_spec_string("imsi",
+      "IMSI",
+      "Internation Mobile Subscriber Identifer",
```

Comment:
Also identifier is missing an I. Wikipedia uses the work "identity", assuming that is the official definition we should probably use that.